src/pages/home/report/reportActionFragmentPropTypes.js

```diff
@@ -25,4 +25,7 @@ export default PropTypes.shape({
 
     // An additional avatar url - not the main avatar url but used within a message.
     iconUrl: PropTypes.string,
+
+    // Fragment edited flag
```

Comment:
```suggestion
    /** Fragment edited flag */
```

Reply:
I think all comments here should be with the same template

Reply:
I think across the App we follow `/** */`. I would've just updated for all the props in the component.